com.github.Ryujinx.json

```diff
@@ -0,0 +1,1266 @@
+{
+    "app-id": "com.github.Ryujinx",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.dotnet"],
+    "command": "start",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
Does it actually work natively on Wayland?

Reply:
No